charts/visor/templates/deployment.yaml

```diff
@@ -55,8 +55,21 @@ spec:
           {{- if .Values.logLevelNamed }}
           - --log-level-named={{ .Values.logLevelNamed }}
           {{- end }}
+
```

Comment:
Nit: This will add whitespace to the values output and makes for noisy diffs in downstream processes. If we were separating logical groupings of options, then it might make sense. If we want whitespace within the template for organization sake, I'd prefer that it's within a comment block.